README.md

```diff
@@ -168,14 +168,6 @@ kubectl delete --ignore-not-found customresourcedefinitions \
 
 `make test-unit`
 
-#### Running *end-to-end* tests on local minikube cluster:
```

Comment:
I don't think we should remove those instructions as those are still valied. Maybe let's just change order and put `kind` info before minikube one, wdyt?

Reply:
Makes sense.